x-pack/plugins/siem/public/store/timeline/epic.ts

```diff
@@ -294,7 +302,9 @@ export const convertTimelineAsInput = (
 ): TimelineInput =>
   Object.keys(timelineInput).reduce<TimelineInput>((acc, key) => {
     if (has(key, timeline)) {
-      if (key === 'kqlQuery') {
+      if (key === 'timelineType') {
```

Comment:
I think if you have added this key into `timelineInput `, and no need to change the value, we can leave it handled by line 375